pkg/webhook/cspc.go

```diff
@@ -342,16 +350,18 @@ func (wh *webhook) validateCSPCUpdateRequest(req *v1beta1.AdmissionRequest) *v1b
 		return response
 	}
 
-	bdr := NewBlockDeviceReplacement().WithNewCSPC(&cspcNew).WithOldCSPC(cspcOld)
+	pOps := NewPoolOperations().WithNewCSPC(&cspcNew).WithOldCSPC(cspcOld)
```

Comment:
do we have check that no two poolspecs should be on same node?

Reply:
Yes, https://github.com/openebs/maya/blob/35af198f6c25f87cd9ab39e6f9cbc34ac154e31f/pkg/webhook/cspc.go#L162